examples/plot_2_multidim_sigs.py

```diff
@@ -0,0 +1,249 @@
+
+for idx in range(epochs):
+
+    if idx % 2 == 0:
+        # Fixation epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.5)}
+    else:
+        # Task epoch
+        sim_components = {'sim_powerlaw': dict(exponent=-2),
+                          'sim_bursty_oscillation': dict(freq=10, cycle='asine',
+                                                         enter_burst=0.75, rdsym=0.75)}
+
+    sigs[idx] = sim_combined(n_seconds, fs, components=sim_components)
+
+####################################################################################################
+
+# Compute features
+thresholds = dict(amp_fraction_threshold=0., amp_consistency_threshold=.5,
+                  period_consistency_threshold=.5, monotonicity_threshold=.8,
+                  min_n_cycles=2)
+
+compute_kwargs = {'burst_method': 'cycles', 'threshold_kwargs': thresholds}
+
+df_features = compute_features_2d(sigs, fs, f_range, axis=None,
+                                  compute_features_kwargs=compute_kwargs)
+
+####################################################################################################
+
+# Add a column specifying epoch type
+for idx, df in enumerate(df_features):
+    df['epoch'] = 'fixation' if idx % 2 == 0 else 'task'
+
+# Merge into a single dataframe
+df_concat = pd.concat(df_features, axis=0, ignore_index=True)
+
+# Limit to bursting cycles
+df_bursts = df_concat[df_concat['is_burst'] == True]
+
+# Plot
+plot_feature_categorical(df_bursts, 'time_rdsym', group_by='epoch', ylabel='Rise-Decay Symmetry',
+                         xlabel=['Fixation', 'Task'])
+
+####################################################################################################
+#
+# Here we show what would happen if axis = 0 was incorrectly used when computing the features
+# for an (n_epochs, n_timeseries) array. If bursts exist at the edges of epochs, they will be
+# incorrectly labeled as non-burst cycles. This is due to the inability to compute consistency
+# features for the first and last cycle of each signal.
+
+df_axis0 = compute_features_2d(sigs, fs, f_range, axis=0, compute_features_kwargs=compute_kwargs)
+df_concat_axis0 = pd.concat(df_axis0, axis=0, ignore_index=True)
+df_bursts_axis0 = df_concat_axis0[df_concat_axis0['is_burst'] == True]
+
+print("""
+    Number of bursts (axis = None): {n_bursts_axis_None}
+    Number of bursts (axis = 0   ): {n_bursts_axis_0}
+""".format(n_bursts_axis_None=len(df_bursts), n_bursts_axis_0=len(df_bursts_axis0)))
+
+####################################################################################################
+#
+# Example 2. 2D Array (n_copies, n_timepoints)
+# --------------------------------------------
+# Features for a 2D array with an (n_copies, n_timepoints) organization will be computed
```

Comment:
You're section-opening sentences read as slightly hard to parse to me. Like, they feel almost like they are missing works (shorthand), though I think what it actually is is that the verb (compute) & object (features) are super far apart, which can make it slightly hard to track, unless one already knows what's being said. 

Like, here I'd write it something like:
"In this example, we will _compute_ features for a 2D array in which we have organized multiple copies of the same signal, to explore applying different thresholds"